index.js

```diff
@@ -51,11 +51,11 @@ let dir = argv.dir
 let input = argv._ || argv.input
 let output = argv.output
 
-let options = {
-  parser: argv.parser || false,
-  syntax: argv.syntax || false,
-  strigifier: argv.strigifier || false,
-  map: argv.map || false
+let defaultOptions = {
```

Comment:
Could use `const` here, since it's renamed and no reassignment through `Object.assign` @L115 takses place anymore, maybe rename to `defaults`  which is cleaner imho. Thats also candidate for being the starting point for `context` but that comes later 😛 .